mirdata/rwc_popular.py

```diff
@@ -0,0 +1,374 @@
+
+    @utils.cached_property
+    def vocal_instrument_activity(self):
+        return _load_voca_inst(utils.get_local_path(
+                self._data_home, self._track_paths['voca_inst'][0]))
+
+
+def download(data_home=None, force_overwrite=False):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, DATASET_DIR, 'annotations')
+    metadata_path = os.path.join(save_path, DATASET_DIR)
+
+    # Downloading multiple annotations
+    for ANNOTATIONS_REMOTE in [ANNOTATIONS_REMOTE_1, ANNOTATIONS_REMOTE_2,
+                               ANNOTATIONS_REMOTE_3, ANNOTATIONS_REMOTE_4]:
+
+        # if exists(data_home) and not force_overwrite:
+        #     return
+
+        if force_overwrite:
+            utils.force_delete_all(ANNOTATIONS_REMOTE, dataset_path=None, data_home=data_home)
+
+        download_path = utils.download_from_remote(
+            ANNOTATIONS_REMOTE, data_home=data_home, force_overwrite=force_overwrite
+        )
+
+        if not os.path.exists(dataset_path):
+            os.makedirs(dataset_path)
+
+        utils.unzip(download_path, dataset_path, cleanup=True)
+
+    missing_files, invalid_checksums = validate(dataset_path, data_home)
+    if missing_files or invalid_checksums:
+        print("""
+            Unfortunately the audio files of the RWC-Popular dataset are not available
+            for download. If you have the RWC-Popular dataset, place the contents into a
+            folder called RWC-Popular with the following structure:
+                > RWC-Popular/
+                    > annotations/
+                    > audio/rwc-p-m0i with i in [1 .. 7]
+                    > metadata-master/
+            and copy the RWC-Popular folder to {}
+        """.format(save_path))
+
+    # metadata
+    download_path = utils.download_from_remote(
+            METADATA_REMOTE, data_home=dataset_path, force_overwrite=force_overwrite)
+    utils.unzip(download_path, metadata_path, cleanup=True)
+
+
+def exists(data_home=None):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, DATASET_DIR)
+    return os.path.exists(dataset_path)
+
+
+def validate(dataset_path, data_home=None):
+    missing_files, invalid_checksums = utils.validator(
+        INDEX, data_home, dataset_path)
+    return missing_files, invalid_checksums
+
+
+def track_ids():
+    return list(INDEX.keys())
+
+
+def load(data_home=None):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, DATASET_DIR)
+    validate(dataset_path, data_home)
+    rwc_popular_data = {}
+    for key in track_ids():
+        rwc_popular_data[key] = Track(key, data_home=data_home)
+    return rwc_popular_data
+
+
+def _load_sections(sections_path):
+    if not os.path.exists(sections_path):
+        raise OSError('Could not find RWC Popular section annotations.')
+    begs = []
+    ends = []
+    secs = []
+
+    if os.path.exists(sections_path):
```

Comment:
you don't need this `if` statement, since you're raising an OSError when it fails. What do you want the behavior to be - if the sections path doesn't exist, should it error or should it return None?